installer files/SimpleIpConfig.iss

```diff
@@ -3,14 +3,20 @@
 #define MyFriendlyVer "2.9.4"
 #define MyAppVer "2.9.4.0"
 #define MyPath ""
+#define MyAppCopyright ""
 
 [Setup]
 AppName={#MyAppName}
 AppVersion={#MyAppVer}
-VersionInfoVersion={#MyAppVer}
+AppVerName={#MyAppName} {#MyAppVer}
+VersionInfoDescription={#MyAppName} installer
+VersionInfoProductName={#MyAppName} {#MyAppVersion}
+VersionInfoVersion={#MyAppVer}.0
+SetupIconFile={app}\{#MyAppName}.exe
```

Comment:
This line gives an error. I think this must be a .ico file. I tried using the icon.ico file but couldn't get the path worked out.

Reply:
Please remove the line
SetupIconFile={app}\{#MyAppName}.exe

Please add your ciopyright name (instead xxx in
#define MyAppCopyright "xxxx"

Pleaswe remove if not necessary the variables.

Reply:
@KurtisLiggett 

I updated the .iss file.
Please check

Reply:
I still cannot get your latest file to run. I get multiple errors.

Reply:
I still cannot get your latest file to run. I get multiple errors.